examples/02-HFSS/HFSS_eigenmode.py

```diff
@@ -0,0 +1,147 @@
+"""
+HFSS: Eigenmode Filter
+--------------------
+This example shows how you can use PyAEDT to automate eigenmode solver in HFSS.
+Eigenmode analysis can be applied to open, radiating structures
+using an absorbing boundary condition. This type of analysis can be very helpful
+to determine the resonant frequency of a geometry or an antenna and can be used to refine
+the mesh at the resonance, even when the resonant frequency of the antenna is not known.
+
+The challenge posed by this method is to identify and filter the non-physical modes
+resulting from reflection from boundaries of the main domain.
+Since the Eigenmode solver sorts by frequency and does not filter on the
+quality factor, these virtual modes will be present when the eigenmode approach is
+applied to nominally open structures.
+When looking for resonant modes over a wide frequency range for nominally
+enclosed structures, several iterations may be required as the minimum frequency
+is determined manually and simulations shall re-run until the complete frequency range is covered
+and all the important physical modes are calculated.
+
+The following script finds the physical modes of a model in a wide frequency range by automating the solution setup.
+During each simulation a user-defined number of modes are simulated and the modes with a Q higher than a user defined value are filtered.
+The next simulation will automatically continue to find modes having a frequency higher than the last mode of tge previous analysis.
+This will continue until the maximum frequency in the desired range is achieved.
+"""
+
+###############################################################################
+# Perform required imports
+# ~~~~~~~~~~~~~~~~~~~~~~~~
+# Run through each cell. This cell imports the required packages.
+
+import sys
+import os
+import pyaedt
+
+# create temporary folder to download the example to this folder.
+
+temp_folder = pyaedt.generate_unique_folder_name()
+project_path = pyaedt.downloads.download_file("eigenmode", "emi_PCB_house.aedt", temp_folder)
+
+
+###############################################################################
+# Launch AEDT
+# ~~~~~~~~~~~
+# Launch AEDT 2023 R1 in graphical mode.
+
+desktopVersion = "2023.1"
```

Comment:
Just to follow Python snake_case convention.
```suggestion
desktop_version = "2023.1"
```